src/pages/home/ReportScreen.js

```diff
@@ -136,10 +136,10 @@ class ReportScreen extends React.Component {
     }
 
     componentWillUnmount() {
-        clearTimeout(this.loadingTimerId);
-        if (window.visualViewport) {
-            window.visualViewport.removeEventListener('resize', this.viewportOffsetTop);
+        if (!window.visualViewport) {
+            return;
         }
+        window.visualViewport.removeEventListener('resize', this.viewportOffsetTop);
     }
```

Comment:
The linter asked me to do a return early here.

Reply:
heads up there's an open pr where this stuff is changing :D https://github.com/Expensify/App/pull/10208

Reply:
yeah I merged that one already